ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -425,9 +423,20 @@ def write_users(self, record: OMetaUserProfile):
                     team_entity = self.metadata.get_by_name(
                         entity=Team, fqdn=str(team.name.__root__)
                     )
+                    if not team_entity:
+                        raise APIError(
+                            error={
+                                "message": "Creating a new team {}".format(
+                                    team.name.__root__
+                                )
+                            }
+                        )
+                    team_ids.append(team_entity.id)
                 except APIError:
                     team_entity = self._create_team(team)
-                team_ids.append(team_entity.id)
+                    team_ids.append(team_entity.id)
+                except Exception as err:
+                    logger.error(err)
```

Comment:
Let's update this to `logger.debug`

Reply:
we only keep the tracebacks in debug mode, we keep errors in logger.error